scripts/ci/precheck_header.sh

```diff
@@ -1,26 +1,10 @@
 #!/usr/bin/env bash
 
-set -exv
```

Comment:
I'm a fan of including `v` here. I think it gives more context when reading through logs when there was a failure. Including it may especially help folks learn how to run the same checks locally without relying on CI.

Reply:
Looks like this is fixed in 52dae4e, thanks!